v63004/outref/gtm8874.txt

```diff
@@ -0,0 +1,270 @@
+# Testing VIEW "[NO]STATSHARE"[:<region-list>]
+#### testA ####
+# enables or disables database statistics sharing for listed regions which permit such sharing. 
+#
+# Without the region-list, the command acts on all regions enabled for sharing. 
+#
+# When a targeted region has sharing disabled, STATSHARE has no immediate effect, 
+# but may cause the process to start sharing when MUPIP SET -STATS enables sharing for that region
```

Comment:
I think lines 7-8 are tested in testB1/testB2 and not in testA. If that is correct, can they be removed here.